sdk/storage/AzureStorageBlob/Source/Data/BlobTransfer+CoreDataProperties.swift

```diff
@@ -29,23 +29,23 @@ import CoreData
 import Foundation
 
 extension BlobTransfer {
-    @nonobjc internal class func fetchRequest() -> NSFetchRequest<BlobTransfer> {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<BlobTransfer> {
         return NSFetchRequest<BlobTransfer>(entityName: "BlobTransfer")
     }
 
-    @NSManaged public internal(set) var destination: URL?
-    @NSManaged internal var endRange: Int64
-    @NSManaged public internal(set) var error: Error?
-    /// The unique identifier for this transfer operation.
-    @NSManaged public internal(set) var id: UUID
```

Comment:
@bsiegel how did you get these `internal(set)` annotations?  Did you just manually edit?

Reply:
Yep, I did. Unfortunately I don't know any way to hint to the codegen how you want the access defined for CoreData properties.

Reply:
No problem. I thought there was some "magic" in the code gen that necessitated you needing to regenerate the code through XCode to make properties accessible. Seems that's not the case, so I think it is fine to manually update these.